node/chainSimulator/components/cryptoComponents_test.go

```diff
@@ -76,6 +78,18 @@ func TestCreateCryptoComponents(t *testing.T) {
 		require.Nil(t, comp.Create())
 		require.Nil(t, comp.Close())
 	})
+	t.Run("should work with bypass blocks sig check", func(t *testing.T) {
```

Comment:
Can we actually check that `fmt.Sprintf("%T")` of the block signer contains `disabled` ?

Reply:
added